da/celestia/celestia.go

```diff
@@ -71,7 +77,17 @@ func (c *DataAvailabilityLayerClient) Stop() error {
 
 // SubmitBlock submits a block to DA layer.
 func (c *DataAvailabilityLayerClient) SubmitBlock(ctx context.Context, block *types.Block) da.ResultSubmitBlock {
-	blob, err := block.MarshalBinary()
+	data, err := block.MarshalBinary()
+	if err != nil {
+		return da.ResultSubmitBlock{
+			BaseResult: da.BaseResult{
+				Code:    da.StatusError,
+				Message: err.Error(),
+			},
+		}
+	}
+
+	blockBlob, err := blob.NewBlob(c.namespace, data, 0)
```

Comment:
use `ShareVersionZero` or `DefaultShareVersion` defined in `celestia-openrpc` and import it here, instead of using constant `0`

Reply:
Using `blob.NewBlobV0` instead.